e2e/app/create-account-page.ts

```diff
@@ -129,27 +127,37 @@ export default class CreateAccountPage extends BasePage {
 
   // select Education Level from a dropdown
   async selectEducationLevel(selectTextValue: string) {
-    const dropdown = new SelectComponent(this.page, FIELD_LABEL.EDUCATION_LEVEL);
+    const dropdown = new SelectComponent(this.page, FIELD_LABEL.EDUCATION_LEVEL, 2);
     await dropdown.select(selectTextValue);
   }
 
   // select Year of Birth from a dropdown
   async selectYearOfBirth(year: string) {
-    const dropdown = new SelectComponent(this.page, FIELD_LABEL.YEAR_OF_BIRTH);
+    const dropdown = new SelectComponent(this.page, FIELD_LABEL.YEAR_OF_BIRTH, 2);
     await dropdown.select(year);
   }
 
   // Combined steps to make test code cleaner and shorter
 
-  // Step 1: Enter Invitation key
+  // Step 1: Fill out Institution
```

Comment:
nit: Fill out institution details - to clarify its just not institution name but email and role as well

Reply:
👍